packages/primitives/src/ArticleOrderedList.stories.tsx

```diff
@@ -301,3 +301,18 @@ export const WithParagraphs: StoryFn = () => (
     </li>
   </OrderedList>
 );
+
+export const ListsBelowEachOther: StoryFn = () => (
+  <>
+    <OrderedList>
+      <li>Listepunkt 1</li>
+      <li>Listepunkt 2</li>
+      <li>Listepunkt 3</li>
+    </OrderedList>
+    <OrderedList variant="letters">
+      <li>Listepunkt 1</li>
+      <li>Listepunkt 2</li>
+      <li>Listepunkt 3</li>
+    </OrderedList>
+  </>
+);
```

Comment:
La til eksempel for å se alignment mellom to lister som ligger etter hverandre, jeg fjerner den når PR'en er godkjent om ingen har noen problemer med det 🥇